rotkehlchen/tests/api/test_eth2.py

```diff
@@ -297,7 +297,7 @@ def test_query_eth2_deposits_details_and_stats(rotkehlchen_api_server, ethereum_
 def test_eth2_add_eth1_account(rotkehlchen_api_server):
     """This test uses real data and tests that adding an ETH1 address with
     ETH2 deposits properly detects validators"""
-    new_account = '0xa966B0eabCD717fa28Bd165F1cE160E7057FA369'
+    new_account = '0xa966B0eabCD717fa28Bd165F1cE160E7057FA369'  # exited
```

Comment:
No, it fails. I stopped looking into it after I saw it was skipped in the CI since it is a slow test and the ultimate goal was to re-enable the windows nightly tests in the CI. I can fix it as part of this PR but maybe it would be better to leave it for another PR? A bunch of the tests that are skipped in the CI fail

Reply:
Oh had not even noticed it's skipped in CI. Yes ofc, if something is skipped in CI ignore for now.